VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-room-shared/waiting-room-base.component.ts

```diff
@@ -1074,6 +1075,7 @@ export abstract class WaitingRoomBaseDirective {
             this.showVideo = false;
             this.showConsultationControls = false;
             this.isPrivateConsultation = false;
```

Comment:
I think we should add a new function called handleNotConnected that sets these values and override it in the judge to call the super.handleNotConnected() and also this.hostWantsToJoinHearing = false